### PR TITLE
Issue #147: Added git workflow guide

### PR DESCRIPTION
**Pull request by Rushi98**
Issue #147 
1. Added git workflow guide.
2. Modified `README.md`, `.github/CONTRIBUTING.md`, `.github/PULL_REQUEST_TEMPLATE.md` to solve #115

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,12 +10,14 @@
 [![GitHub contributors](https://img.shields.io/github/contributors/MobileApplicationsClub/ARD.svg)](https://github.com/MobileApplicationsClub/ARD/graphs/contributors)
 [![Slack](https://img.shields.io/badge/Slack-join%20chat-brightgreen.svg)](https://join.slack.com/macbitsgoa/shared_invite/MjAxNDMzNzM0NjEwLTE0OTgwNjc4NDAtMmVmMWQyZjk2MA)
 
-This app runs on Android OS version above and including Lollipop (*API 21*) uptil Nougat (*API 25*).
+This app runs on Android OS version above and including Lollipop (*API 21*).
```
Comment (Rushi98) [inline README.md]: Our app should run on oreo.

File: .github/Git workflow.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+0. Don't make changes to `master` branch.
```
Comment (kukreja-vikramaditya) [inline .github/Git workflow.md]: Suggestion: avoid using space in file name. Consider using `-` all  with lower case

File: .github/Git workflow.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+git merge upstream/master
+```
+
+_or_ if you failed at 0
+
+```bash
+git reset --hard upstream/master
+```
+
+5. Do
```
Comment (kukreja-vikramaditya) [inline .github/Git workflow.md]: To many dos don't you think?  :rofl:

File: README.md [hunk 4]
```diff
@@ -46,31 +48,27 @@ git clone https://github.com/yourusername/ARD.git
...
-5. Make your changes and commit them
-```bash
-git add .
-git commit -m "Give commit message"
-```
+### Contributing
+See `.github/CONTRIBUTING.md`.
 
-6. Update your fork
+If you're not proficient with **git**, it'll be better if you follow `.github/Git workflow.md`.
```
Comment (kukreja-vikramaditya) [inline README.md]: name change of file

**Pull request merged**